.docker/app/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM php:8.0.3-cli-alpine3.12
+
+WORKDIR /
+
+RUN apk update && apk add bash
+
+ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
+
+RUN chmod +x /usr/local/bin/install-php-extensions && sync && \
+    install-php-extensions @composer gd pdo pdo_mysql
+
+COPY . /app
+COPY ./.docker/app/php.ini /usr/local/etc/php/conf.d/php.ini
+
+WORKDIR /app
+RUN composer install && chmod 640 *
+
+ENV ENVIRONMENT=DEVELOPMENT
```

Comment:
What is the purpose of this variable? I can't see it being used anywhere.

Reply:
Good practice for future